packages/ember-testing/lib/helpers.js

```diff
@@ -111,8 +111,16 @@ asyncHelper('fillIn', fillIn);
   var $el = find('.my-selector');
   ```
 
+  With the `context` param:
+
+  ```javascript
+  var $el = find('.my-selector', 'parent-element');
```

Comment:
Here and below: Perhaps `'.parent-element-class'`? Using a custom element in an example like this seems a bit cutting edge.